server/app/views/admin/questions/QuestionEditView.java

```diff
@@ -457,10 +468,94 @@ private DomContent buildUniversalQuestion(QuestionForm questionForm) {
+                        "You cannot edit this setting since the question is not a universal"
+                            + " question and because this property is already set on a question"
+                            + " named %s.",
+                        otherQuestionName);
+                String initialAlertText =
+                    questionForm.isUniversal()
+                        ? alreadySetAlertText
+                        : nonUniversalAlreadySetAlertText;
+                tagSubsection =
+                    tagSubsection
+                        .with(
+                            ViewUtils.makeAlertInfoSlim(
+                                initialAlertText,
+                                /* hidden= */ false,
+                                /* classes...= */ "cf-primary-applicant-info-alert",
+                                "usa-alert-primary-applicant-info"))
+                        .withData("already-set-alert", alreadySetAlertText)
+                        .withData(
+                            "non-universal-already-set-alert", nonUniversalAlreadySetAlertText);
+              } else {
+                // Include both the toggle and the alert. We'll show/hide each based on the state
+                // of the universal toggle in admin_question_edit.ts.
+                tagSubsection =
+                    tagSubsection
+                        .with(
+                            p().withClasses("px-1", "pb-2", "text-sm", "text-gray-600")
+                                .with(
+                                    span(primaryApplicantInfoTag.getDescription()),
+                                    ViewUtils.makeToggleButton(
+                                        /* fieldName= */ primaryApplicantInfoTag.getFieldName(),
+                                        /* enabled= */ questionForm
+                                            .primaryApplicantInfoTags()
+                                            .contains(primaryApplicantInfoTag),
+                                        /* hidden= */ !questionForm.isUniversal(),
+                                        /* idPrefix= */ Optional.of(
+                                            primaryApplicantInfoTag.getFieldName()),
+                                        /* text= */ Optional.of(
+                                            primaryApplicantInfoTag.getDisplayName()))))
+                        .with(
+                            ViewUtils.makeAlertInfoSlim(
+                                "You cannot edit this setting since the question is not a universal"
+                                    + " question.",
+                                /* hidden= */ questionForm.isUniversal(),
+                                /* classes...= */ "cf-primary-applicant-info-alert",
+                                // This removes the top 1rem margin built in to the USWDS component
+                                "usa-alert-primary-applicant-info"));
```

Comment:
*thought, non-blocking*: what naming convention do we want to introduce for classes that modify USWDS styles? Do we want to go in a tailwind-ish direction and name it something like `clear-usa-alert-top-margin`?